### ISSUE TITLE
Loss of Object detection model using ssd_mobilenet_v2_quantized_300x300_coco increases after every 10k-12k steps

### ISSUE THREAD

**Issue by rhlr** [tensorflow/models#6334]
------------------------

### System information
- **What is the top-level directory of the model you are using**:
  ~/models/research
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**:
  No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**:
  Linux instance-3 4.15.0-1028-gcp #29~16.04.1-Ubuntu SMP Tue Feb 12 16:31:10 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
  VERSION="16.04.5 LTS (Xenial Xerus)"
  VERSION_ID="16.04"
  VERSION_CODENAME=xenial

- **TensorFlow installed from (source or binary)**:
  binary
- **TensorFlow version (use command below)**:
  tf.VERSION = 1.12.0
  tf.GIT_VERSION = v1.12.0-0-ga6d8ffae09
  tf.COMPILER_VERSION = v1.12.0-0-ga6d8ffae09
- **Bazel version (if compiling from source)**:
- **CUDA/cuDNN version**:
  9.0/7.4
- **GPU model and memory**:
  NVIDIA TESLA K80: 11439MiB
- **Exact command to reproduce**:
`python object_detection/model_main.py --pipeline_config_path=ssd_mobilenet_v2_quantized_300x300_coco.config  --model_dir=train  --num_train_steps=500000 --alsologtostder`

### Describe the problem
I'm re-training `ssd_mobilenet_v2_quantized_300x300_coco` `object detection` model on a custom dataset. The dataset consists of approx 2.6k images and 19 classes. After the training step reaches 10k-12k the loss graph starts increasing. This happens even if I change my model to `ssd_mobilenet_v2_coco` and at the same step range. I couldn't find anything that is related to this behaviour in the config file. Also this disappers when using `faster_rcnn` models. When the issue arises the `mAP` becomes almost constant. Also tha accuracy doesn't go beyond 50%. Can anyone explain this behaviour ?

### Source code / logs
![Screenshot from 2019-03-10 10-15-53-02](https://user-images.githubusercontent.com/1620866/54081184-67c08800-4326-11e9-9e1e-3624cdc13e8f.png)
a) ssd_mobilenet_v2_quantized_300x300_coco
![Screenshot from 2019-03-10 10-17-26-01](https://user-images.githubusercontent.com/1620866/54081185-6db66900-4326-11e9-9009-31e3c088b6ae.png)
b) ssd_mobilenet_v2_coco

### Sample dataset:

![individualImage](https://user-images.githubusercontent.com/1620866/54081211-e3bad000-4326-11e9-8a0b-3c993252ed3e.png)

### Config

[ssd_mobilenet_v2_quantized_300x300_coco.txt](https://github.com/tensorflow/models/files/2949100/ssd_mobilenet_v2_quantized_300x300_coco.txt)
a) ssd_mobilenet_v2_quantized_300x300_coco.config

[ssd_mobilenet_v2_coco.txt](https://github.com/tensorflow/models/files/2949102/ssd_mobilenet_v2_coco.txt)
b) ssd_mobilenet_v2_coco.config

**Comment by rootkitchao** [comment]
By checking your configuration file, I think there is a problem with the hyperparameters you set.
'learning_rate: {
        exponential_decay_learning_rate {
          initial_learning_rate: 0.004
          decay_steps: 5000
          decay_factor: 0.50
        }
      }'
You set a too small decay_factor and wrong decay_steps.According to your settings, the learning rate will drop rapidly, and then continue training at a very low learning rate after 5000 steps.

**Comment by rhlr** [comment]
Even if I used the default setting, the behaviour was exactly same. The default setting in the config was:

`learning_rate: {
exponential_decay_learning_rate {
initial_learning_rate: 0.004
decay_steps: 800720
decay_factor: 0.95
}
}`

**Comment by rootkitchao** [comment]
The default configuration may be for multi-GPU systems.It is recommended to try the initial learning rate = 0.1, 0.01. Or use the SSD Mobilenet V2 FPN model.

**Comment by rootkitchao** [comment]
I noticed that you are using NVIDIA Tesla K80.NVIDIA Tesla K80 actually has two GK210 GPUs.I'm not sure if the default command can correctly use two GPUs simultaneously.I suggest you try the following commands:
`python legacy/train.py --logtostderr --num_clones=2 --ps_tasks=1 --train_dir=D:\tf_project\plate_detector\model\train --pipeline_config_path=D:\tf_project\plate_detector\model\ssd_mobilenet_v2_fpn_320.config`
This may speed up your training.

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
